corehq/apps/case_importer/views.py

```diff
@@ -226,6 +226,9 @@ def excel_fields(request, domain):
     if search_column in excel_fields:
         excel_fields.remove(search_column)
 
+    if 'domain' in excel_fields and DOMAIN_PERMISSIONS_MIRROR.enabled(domain):
```

Comment:
Can you add a test for a domain that doesn't have the flag enabled but does include a `domain` column, and then validates that a `domain` case property was correctly created/updated?

This commit itself looks fine, but I'd expect [this check](https://github.com/dimagi/commcare-hq/pull/28560/files#diff-210f4f2b3c4c9a158074ce12c9340073c0a32fd6fb87cce297d7a5ca5a357f48R155) to throw an error if a domain *without* the flag on uses the column. I'd also think [this check](https://github.com/dimagi/commcare-hq/pull/28560/files#diff-c4b45614676616b2f3694a509e3cd0448b05126f224bb34b799d3044d01e3274R87) could cause rows to be skipped.